# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/linux/sifive-fans.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-5.13/arch/riscv/boot/dts/sifive/hifive-unmatched-a00.dts.vanilla	2021-07-20 15:43:29.828968866 +0200
+++ linux-5.13/arch/riscv/boot/dts/sifive/hifive-unmatched-a00.dts	2021-07-20 15:47:26.605981081 +0200
@@ -70,6 +70,24 @@
 			max-brightness = <255>;
 			linux,default-trigger = "none";
 		};
+
+                fan1 {
+                        label = "fan1";
+                        color = <LED_COLOR_ID_BLUE>;
+                        pwms = <&pwm1 2 7812500 PWM_POLARITY_INVERTED>;
+                        active-low = <1>;
+                        max-brightness = <255>;
+                        linux,default-trigger = "none";
+                };
+
+                fan2 {
+                        label = "fan2";
+                        color = <LED_COLOR_ID_BLUE>;
+                        pwms = <&pwm1 3 7812500 PWM_POLARITY_INVERTED>;
+                        active-low = <1>;
+                        max-brightness = <255>;
+                        linux,default-trigger = "none";
+                };
 	};
 
 	hfclk: hfclk {
